vue/components/ui/molecules/input-image/input-image.vue

```diff
@@ -3,57 +3,57 @@
         class="input-image"
         v-bind:multiple="multiple"
         v-bind:accept="accept"
+        ref="uploadArea"
         v-on:update:files="loadFiles"
     >
-        <template v-slot="{ openModal }">

```

Comment:
Can't you keep providing the `openModal` in the slot so that these changes aren't needed?